test/controllers/users/invitations_controller_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class Users::InvitationsControllerTest < ActionController::TestCase
+  test "admin can access invite page" do    
+    user = User.create(:email => "example@example.com", :password => "password", :password_confirmation => "password", :admin => true)
+    sign_in user
+    get :new
```

Comment:
Is this controller provided courtesy of the `devise_invitable` gem?  If it is, we can go ahead and assume the code works (in general you don't want to spend time testing things that gems provide).